.github/workflows/dev-workflow.yml

```
@@ -29,18 +29,18 @@ jobs:
           components: clippy, rustfmt
           target: wasm32-unknown-unknown
           toolchain: stable
-
+      - uses: cargo-bins/cargo-binstall@main
       - name: Cache builder
         id: cache-builder
         uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)